bot/seasons/valentines/savethedate.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+
+log = logging.getLogger(__name__)
+
+emoji = [":heart:", ":couple_with_heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
```

Comment:
We'd prefer all-caps for module-level constants. A more descriptive variable name might be appropriate as well.